src/libappimage/config.h.in

```diff
@@ -0,0 +1,13 @@
+// LibAppImage Version
+
+#define LIBAPPIMAGE_VERSION_MAJOR @V_MAJOR@
+#define LIBAPPIMAGE_VERSION_MINOR @V_MINOR@
+#define LIBAPPIMAGE_VERSION_PATCH @V_PATCH@
+#define LIBAPPIMAGE_VERSION_SUFFIX @V_SUFFIX@
+
+#define LIBAPPIMAGE_VERSION_STR "@libappimage_VERSION@"
```

Comment:
Why the `_STR` suffix? Should match what's defined in CMake, i.e., `LIBAPPIMAGE_VERSION`.

Also, do we set either of those manually in CMake somewhere? If yes, it should be removed there to make sure the config header works.

Reply:
> Why the _STR suffix?

Processor defines doesn't have a type, this suffix gives a hint about it.

Reply:
I don't think that's really necessary, people use such vars as documented... I don't like the inconsistency. You can also rename the CMake variable.

Reply:
> Also, do we set either of those manually in CMake somewhere? If yes, it should be removed there to make sure the config header works.

It's used mostly in the test. But yes it should be properly tested, my bad.

Reply:
> people use such vars as documented...

They are not documented. It's a common practice using the `_STR` prefix but it's also true the opposite. If you prefer it without the suffix let's change it.

Reply:
I don't care really. I do see the purpose of the suffix. Therefore the suggestion to just add it in CMake too.

Reply:
Too late. Changing the var name in CMake will require to check/test all it's usages. Let's not touch what is already working.